lib/rust/ensogl/component/scrollbar/src/lib.rs

```diff
@@ -236,6 +246,27 @@ impl Frp {
                 });
 
 
+            // === Click and hold repeated scrolling ===
+
+            background_drag_start <- base_frp.is_dragging_background.on_true();
+            click_and_hold_timer.start <+ background_drag_start.constant(click_and_hold_config);
+            click_and_hold_timer.stop <+ base_frp.is_dragging_background.on_false();
+
+            mouse_pos_at_timer_trigger <- mouse_position.sample(&click_and_hold_timer.on_trigger);
+            offset_from_thumb_px <- mouse_pos_at_timer_trigger.map2(&thumb_center_px,
+                |mouse_pos, thumb_center| thumb_center - mouse_pos.x);
```

Comment:
would `offset_from_thumb_px <- mouse_pos_at_timer_trigger.map2 - thumb_center_px;` not work here? I think it should. If not, we should add it to the macro.

Reply:
That doesn't work, and I would need member access for `thumb_center_px.x`, which makes it more complicated and potentially ambiguous to parse through macros. Also the expression is the wrong way around, though in this specific case changing the sign would be acceptable.